sdk/storage/Azure.ResourceManager.Storage/CHANGELOG.md

```diff
@@ -10,6 +10,18 @@
 
 ### Other Changes
 
+## 1.0.1 (2022-09-23)
```

Comment:
I'm guessing you updated the wrong file, this is storage

Reply:
Thanks for the catch! I have fixed with it with the right changelog now